README.md

```diff
@@ -1,4 +1,4 @@
-<div align="center">
```

Comment:
Either keep the logo in center or move the tag to left as well.

Reply:
Thanks. Accidental removal